styles/Home.module.scss

```
@@ -1,5 +1,54 @@
 @import './variables.module.scss';
 
+.body {
+  font-family: $font;
+}
+
+.header {
+  display: flex;
+  padding: 10px 10%;
+  align-items: center;
+  justify-content:space-between;
+  background-color: $color-primary;
+  border-bottom: 5px solid $color-secondary;
+}
+.nav__links {
+  list-style: none;
+  display: inline-block;
+}
+
+.nav__links li {
+  display: inline-block;
+  padding: 0px 30px;
+  color: $color-white;
+  font-size: 18px;
+  transition: all 0.3s ease 0s;
+}
+
+.nav__links li:hover {
+  color: $color-grey;
+}
+
```

scss umi nestovani, takze to css kolem nav__links muze byt takhle:
```
.nav {
  &__links {
    list-style: none;
    display: inline-block;
    li {
      display: inline-block;
      padding: 0px 30px;
      color: $color-white;
      font-size: 18px;
      transition: all 0.3s ease 0s;
      &:hover {
        color: $color-grey;
      }
    }
  }
}
```

---

Dávám to tam. Jak přesně funguje to &? Něco ve smyslu - dosaď všechny rodiče nade mnou? 

Nešlo by to udělat i takhle bez toho prvního řádku .nav {} ?

```suggestion
.nav__links {
    list-style: none;
    display: inline-block;
    li {
      display: inline-block;
      padding: 0px 30px;
      color: $color-white;
      font-size: 18px;
      transition: all 0.3s ease 0s;
      &:hover {
        color: $color-grey;
      }
    }
  }
```

---

jo, šlo by to. ten & je jak rikas refenrece na rodiče. takze z
```
.nav {
  &__links {}
}
```
bude v css `.nav__links`. Místo & jsem dosadil toho parent (rodiče)

---

dalsi priklad:
scss
```
.nav {
  li {}
}
```
bude v css `.nav li` (select všechny li uvnitr .nav)

---

Rozumím